charts/argo-cd/templates/argocd-configs/argocd-secret.yaml

```diff
@@ -34,10 +34,13 @@ data:
   {{- with .Values.configs.secret.gogsSecret }}
   webhook.gogs.secret: {{ . | b64enc }}
   {{- end }}
+  {{/* Helm v3.13.x needs the if condition. */}}
+  {{- if .Values.configs.secret.azureDevops }}
```

Comment:
Hi @cherrot , @TechDawg270,
Can you please check your environment if this works well ? 🙇

Reply:
Ah no, need to add here. I will fix..
https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/templates/argocd-configs/argocd-secret.yaml#L19

Reply:
works well! sorry for delayed response as i was traveling. thank you for the quick fix